### ISSUE TITLE
Plugin doesn't handle duplicate account names in different partitions

### ISSUE THREAD

**Issue by thenoid** [tilfinltd/aws-extend-switch-roles#133]
In standard and govcloud we have 4 accounts

- Standard
  - AccountA-controlplane
  - AccountA-app
- Govcloud
  - AccountA-controlplane
  - AccountA-app

Because they are on separate partitions, this is perfectly valid.  However, the plugin get's very confused as it appears to have no way to distinguish between partitions and only shows the logins for the Standard Partition.

Any help here would be greatly appreciated.

**Comment by thenoid** [comment]
My config looks like:
- [x] Show only matching roles
```
[hv-controlplane]
aws_account_id = hv-controlplane
target_role_name = XACT-FullAdmin

[hirevue]
aws_account_id = hirevue
role_arn = arn:aws:iam::874344875028:role/XACT-FullAdmin
source_profile = hv-controlplane
color = b2172e

[hirevue-ro]
aws_account_id = hirevue
role_arn = arn:aws:iam::874344875028:role/XACT-ReadOnly
source_profile = hv-controlplane
color = 000000

[hv-controlplane-gov]
aws_account_id = hv-controlplane

[govcloud]
aws_account_id = 806043367460
source_profile = hv-controlplane-gov
role_arn = arn:aws-us-gov:iam::806043367460:role/XACT-FullAdmin


```

**Comment by thenoid** [comment]
Wow, thanks github for stealing my keybindings....let's try this again.
- [x] Hide account id

```
[AccountA-controlplane]
aws_account_id = AccountA-controlplane
target_role_name = XACT-FullAdmin

[AccountA]
aws_account_id = AccountA
role_arn = arn:aws:iam::NUMBERS:role/XACT-FullAdmin
source_profile = AccountA-controlplane
color = b2172e

[AccountA-ro]
aws_account_id = AccountA
role_arn = arn:aws:iam::NUMBERS:role/XACT-ReadOnly
source_profile = AccountA-controlplane
color = 000000

[AccountA-controlplane-gov]
aws_account_id = AccountA-controlplane

[govcloud]
aws_account_id = NUMBERS
source_profile = AccountA-controlplane-gov
role_arn = arn:aws-us-gov:iam::NUMBERS:role/XACT-FullAdmin
```

**Comment by tilfin** [comment]
I don't know about the details about the GovCloud environment. It is not supported that multiple root entries referred by *source_profile* have the same *aws_account_id*. We need to add the attribute to distinguish whether GovCloud or not.

**Comment by tilfin** [comment]
@thenoid Is this issue solved by the feature 'base account with role_name' released at latest version 2.2.0? If you use a different user or role to log in to your regular account and GovCloud one respectively, specify it in **role_name**.
